src/templates/blog-post.tsx

```diff
@@ -104,12 +107,20 @@ const BlogArticleTemplate: React.FC<BlogArticleTemplateProps> = ({
       squareImage={getImage(featuredImageSquared)!}
     />
   );
+  const leadbox = {
+    title: t('office.leadbox.title'),
+    link: t('office.leadbox.link'),
+    linkTo: '/career/',
+    icon: <Astronaut />,
+  };
+
```

Comment:
wrong key, I will suggest the fixed key (`blogpost`) and suggest according changes to the wrong key (being `blogpos`)
```suggestion
  const leadbox = {
    title: t('blogpost.leadbox.title'),
    link: t('blogpost.leadbox.link'),
    linkTo: '/career/',
    icon: <Astronaut />,
  };

```

In the future I think it would be awesome to pass in some unique text per blog post to make the leadbox aware of the surrounding content. Like "have you learned something about gatsby, then join us to learn more" or so.

Reply:
Thanks for your feedback 🙏
I like the idea with the customizable leadbox for blogposts. I will add a corresponding field to the blogpost template